rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -2057,12 +2058,12 @@ void game_list_frame::PopulateGameGrid(int maxCols, const QSize& image_size, con
 /**
 * Returns false if the game should be hidden because it doesn't match search term in toolbar.
 */
-bool game_list_frame::SearchMatchesApp(const std::string& name, const std::string& serial)
+bool game_list_frame::SearchMatchesApp(const QString& name, const QString& serial, const QString& custom_name)
```

Comment:
Passing the custom name here is useless.
It should just be obtained after checking im_search_text.sEmpty below.

Reply:
imo produces clearer code but still a fair point, I guess I "forgot" `SearchMatchesApp` is also a member of `game_list_frame`